src/objects/Trial_Level__mdt.object

```diff
@@ -105,7 +105,4 @@
         <label>Water required</label>
         <type>Checkbox</type>
     </fields>
-    <label>Trial Level</label>
```

Comment:
@Drago404  why did you delete those?

Reply:
When I retrieved the metadata from my org using Workbench that was the result (it didn't include those). Maybe I hadn't formatted my package.xml correctly for the retrieval.

Reply:
Please remove this deletion form a pull request.